pkg/formats/plist.go

```diff
@@ -0,0 +1,105 @@
+package formats
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/alecthomas/chroma/quick"
+	"github.com/go-xmlfmt/xmlfmt"
+	"howett.net/plist"
+)
+
+var (
+	_ Encoding = plistEncoding{}
+	_ Decoder  = &plistDecoder{}
+	_ Encoder  = &plistEncoder{}
+)
+
+type plistEncoding struct{}
+
+func (plistEncoding) NewDecoder(r io.Reader) Decoder {
+	b, err := ioutil.ReadAll(r)
+	if err != nil {
+		return nil
+	}
+	decoder := plist.NewDecoder(bytes.NewReader(b))
+	return &plistDecoder{decoder, false}
```

Comment:
Why ReadAll here? Couldn't this whole function just be:
```go
func (plistEncoding) NewDecoder(r io.Reader) Decoder {
  return &plistDecoder{plist.NewDecoder(r), false}
}
```

Reply:
I think because plist takes a `io.ReadSeeker` and he's using `bytes.NewReader` to do that (since we only provide an `io.Reader`)

Reply:
Yes that is the reason.